benchmark/fio_benchmark_setup.yml

```diff
@@ -60,6 +60,7 @@
     - name: Set application for benchmark pool
       command: "ceph osd pool application enable {{ _pool_name }} rbd"
       run_once: True
+      when: ceph_stable_release != "jewel"
```

Comment:
Do we want this to be '== "luminous" ?

Reply:
I figure when we move to N release it'll still work, rather than having to manually edit to work on luminous + n

Reply:
If rpc-ceph is to be used for hammer upgrades, then it needs to exclude hammer and jewel at the very least.

Reply:
ahh good idea, that should work since we run the ceph-defaults role before hand. ok fix incoming.